rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -3135,8 +3145,14 @@ int LuaUnsyncedRead::GetSoundDevices(lua_State* L)
 
 	for(int i; i < devices.size(); ++i) {
 		std::string &device = devices[i];
-		lua_pushsstring(L, device);
-		lua_rawseti(L, -2, i);
+
+                lua_createtable(L, 0, 1); {
+			lua_pushliteral(L, "name");
+	                lua_pushsstring(L, device);
+	                lua_rawset(L, -3);
+		}
+
+                lua_rawseti(L, -2, i);
```

Comment:
Lua tables start from 1:
```suggestion
                lua_rawseti(L, -2, i + 1);
```